docs/source/projects.rst

```diff
@@ -402,6 +402,29 @@ for your run. For example:
   - You must use a *new cluster* specification when running an MLflow Project on Databricks. Running
     Projects against existing clusters is not currently supported.
 
+Databricks Execution Tips
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When running an MLflow Project on Databricks, the following tips may be helpful.
+
+Using SparkR on Databricks
+##########################
+
+In order to use SparkR in an MLflow Project run on Databricks, your project code must first install
+and import SparkR as follows:
+
+.. code-block:: R
+
+  install.packages("/databricks/spark/R/pkg", repos = NULL)
```

Comment:
Looks good, although I wonder if we should update this code snippet so that it can run successfully both in Databricks & on users' laptops for local iteration (e.g. update the snippet to check if `/databricks/spark/R/pkg` exists & install from there if it does, & just do a normal `install.packages("SparkR")` if not)

Reply:
Good point! Done!